guiiluiz/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script type="text/javascript" src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Lista de Tarefas</title>
+</head>
+
+<body>
+    <div class="container">
+            <h1 class="title">Lista de Tarefas</h1>
+            <h4 class="subtitle">Clique duas vezes em um item para marcá-lo como completo</h4>
+        <div class="content" id="addItem">
+            <input type="text" id="userInput">
```

Comment:
o id poderia ser fazer mais sentido porque o que está sendo inserido é uma task 
```suggestion
            <input type="text" id="taskInput">
```